.github/ISSUE_TEMPLATE/stale_issue_report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Stale issue report
+about: Report issues with using the stale action
+title: ''
+labels: bug
+assignees: ''
+---
+
+<!-- Have you tried the [debugging](https://github.com/actions/stale#debugging) section of the readme? -->
+
+## Describe your issue
+
+## Your stale action configuration
+<!-- You can include a link with the issue or pull request having concerning this issue report. -->
```

Comment:
"having concerning" :/

Reply:
I'll just remove the comment, the header should be clear enough. 😄